Tevling/Pages/Challenges.razor.cs

```diff
@@ -22,10 +24,28 @@ public partial class Challenges : ComponentBase, IDisposable
     private bool _showAllChallenges = true;
 
     private bool _showOutdatedChallenges;
+    private bool _showTimeChallenges = true;
```

Comment:
Once we move to .NET 10 we can get rid of these backing fields 🙂
https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-14#the-field-keyword